src/api-reference/travel-profile/v1.loyalty-program-resource.markdown

```diff
@@ -47,40 +49,38 @@ If the request is sent by a TMC, the request can update any loyalty program for
 
 ### Restrictions
 
-This function is only available to travel suppliers who have completed the [Concur application review process](/manage-apps/app-certification.html). Suppliers may post loyalty membership information for their loyalty programs only.
+This function is only available to travel suppliers who have completed the [SAP Concur application review process](/manage-apps/app-certification.html). Suppliers may post loyalty membership information for their loyalty programs only.
 
 ## Request
 
 ### Headers
 
-#### Authorization header
+#### Authorization Header
 
-Authorization header with OAuth token for valid Concur user. Required.
+Authorization header with OAuth token for valid SAP Concur user. Required.
 
-#### Accept header
+#### Accept Header
 
 application/xml
 
-### Request body
-
-#### Root elements
+### Request Body
 
 This request contains the **LoyaltyMembershipUpdate** parent element with a **Membership** child element for each included loyalty program. The **Membership** element has a **UniqueID** attribute containing the loyalty program identifier, and the following child elements.
```

Comment:
Can you throw these fields into backticks? I wouldn't normally mentioned it but you did it for all the other examples of this in the doc set.

Reply:
Fixed